ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -79,16 +79,16 @@ export default {
       },
     },
     {
-      id: 'DelubrumSav Seeker Baleful Onslaught Stillness',
+      id: 'DelubrumSav Seeker Baleful Onslaught Solo',
       netRegex: NetRegexes.startsUsing({ source: 'Trinity Seeker', id: '5AD6', capture: false }),
       netRegexDe: NetRegexes.startsUsing({ source: 'Trinität Der Sucher', id: '5AD6', capture: false }),
       netRegexFr: NetRegexes.startsUsing({ source: 'Trinité Soudée', id: '5AD6', capture: false }),
       netRegexJa: NetRegexes.startsUsing({ source: 'トリニティ・シーカー', id: '5AD6', capture: false }),
       alertText: (data, _, output) => output.text(),
       outputStrings: {
         text: {
-          en: 'Stillness Tank Cleave',
-          de: 'Stille Tank Cleave',
+          en: 'Solo Tank Cleave',
+          de: 'Solo Tank Cleave',
```

Comment:
```suggestion
          de: 'Solo Tank Cleave',
```

In general, when changing translations, I usually remove everything else so that translators can come back and add a new translation.  (Unless this is proper German?)

Reply:
You could change it to `Solo Tank Cleave` and would understood by 99%.
But if this is just a normal tank cleave, couldn't we just use the response for that (as for other solo cleaves we dont specify the solo part)

Reply:
Use of 'solo' over the same tank cleave puts more emphasis on its distinction from the other tank cleave in the same encounter, plus its near proximity with its first appearance which I think is why the extra word may be preferred. As the translation appears good enough I'll leave it in the PR and next time I'll keep in mind to remove the prior translations.

Up for further discussion if others think the generic response should be used instead.